tests/test_proxy_functional.py

```diff
@@ -15,6 +17,8 @@
 from aiohttp import web
 from aiohttp.client_exceptions import ClientConnectionError
 
+PY_310 = sys.version_info >= (3, 10)
```

Comment:
I think we could have a reusable marker here
```suggestion
PY_310 = sys.version_info >= (3, 10)
xfail_under_py310_except_macos = pytest.mark.xfail(
    PY_310 and platform.system() != "Darwin",
    raises=ConnectionAbortedError,
    reason="The secure proxy fixture does not seem to work under Python 3.10 on Linux or Windows. See https://github.com/abhinavsingh/proxy.py/issues/622.",
)
```
and apply it on tests as `@xfail_under_py310_except_macos`.

Reply:
Also, the best place for the version constant seems to be https://github.com/aio-libs/aiohttp/blob/c9c9bec/aiohttp/helpers.py#L57. Consider moving it there.

Reply:
Done.